src/Controller/ControllerInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace RebelCode\EddBookings\RestApi\Controller;
+
+use RebelCode\EddBookings\RestApi\Resource\ResourceInterface;
+
+/**
+ * An API controller - something that can work with API resources.
+ *
+ * @since [*next-version*]
+ */
+interface ControllerInterface
+{
+    /**
+     * Retrieves resources.
+     *
+     * @since [*next-version*]
+     *
+     * @param array $params The params.
+     *
+     * @return ResourceInterface[] The resource instances.
+     */
+    public function get($params = []);
```

Comment:
I think that this method should declare a `@throws` type, and exceptions inside should get wrapped into that type. This is especially true, because many implementations use `_containerGet()`, which is quite prone to throwing `NotFoundException`. Consumers of `ControllerInterface#get()` have no idea that you are using a container inside.